pkgs/development/python-modules/django-mdeditor/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, django
+}:
+let
+  version = "0.1.20";
+in
+buildPythonPackage {
```

Comment:
You could use
```
buildPythonPackage rec {
  pname = "django-mdeditor";
  version = "0.1.20";

  src = fetchFromGitHub {
    [...]
    rev = "v${version}";
```
instead of `let ... in`

Reply:
you could, I also like it more this way, but if you do it with let in it saves 8 bytes on compilation

Reply:
okay than its fine for me :)